### ISSUE TITLE
[Question]: What happen to our inheritance if your company's service shutdown?! 

### ISSUE THREAD

**Issue by Sami32** [nunchuk-io/nunchuk-android#60]
In the case your company shutdown in the future, for whatever reason, what happen to our inheritance? How can anyone can claim it if your service is down???

I apology if i missed this information in your documentation.

**Comment by hugohn** [comment]
Good question. Let me rephrase it: What happens to the inheritance service if Nunchuk ceases to exist?

The short answer is that just like other types of services, if the business goes away the services will no longer be available. What we recommend to our users to mitigate unexpected events (such as Nunchuk ceasing to exist) is to use a "**rolling short timelock**" approach: Set an Activation Date not too far in the future, such as one or two years from today. Then the owner can review the inheritance plan every year and push the date back if necessary.

To help remind users, we start sending email notifications as the timelock is about to expire, starting at the 6-month mark. This approach allows users to lengthen the time the inheritance can be claimed as needed, while guarding against long-term uncertainties. If something unfortunate happens to Nunchuk, the owner can use their own keys to move funds to a new wallet. If something unfortunate happens to the owner, their beneficiaries can claim the inheritance quickly without having to wait many years.

Since the odds that Nunchuk ceasing to exist _and_ the owner passing away within 1-2 years is small, the approach above reduces risks significantly.

**Comment by Sami32** [comment]
This kind of information should be added to your FAQ IMHO, because we never know when we will die, affected by amnesia, Alzheimer or whatever and companies cease to exist more than ever recently.
As uncertainty is not compatible for a inheritance plan for me i consider this information very importante.
